### PR TITLE
enable period h_mpo init

### PR DESCRIPTION
**Pull request by jiangtong1000**
Now we can assign the `mol_list` 's`period` whether to init mpo with boundary condition

### CODE REVIEW

File: renormalizer/cv/spectra_cv.py [hunk 1]
```diff
@@ -49,22 +50,20 @@ def check_qn(self, X):
         print('Quantum number of X', check_1)
     '''
     def run(self):
-        try:
+        if importlib.util.find_spec("cupy"):
```
Comment (liwt31) [inline renormalizer/cv/spectra_cv.py]: Could you explain why `forkserver` is necessary for `cupy`? I'm sorry but I'm not familiar with this feature.

Comment (jiangtong1000) [reply renormalizer/cv/spectra_cv.py]: I found the same problem and solution at [pytorch/pytorch/issues/334] and [chainer/chainer/issues/1087], my understanding is that before starting multiprocessing, the cuda has already been initialized, so actually each process did't inherent the cuda resource, that means cuda should be initialized at each process separately by setting the start method as `spwan` or `forkserver`

Comment (liwt31) [reply renormalizer/cv/spectra_cv.py]: Thanks for the explanation.

**Comment by liwt31** [comment]
I'm OK this PR but as you've requested the review from jjren I'll leave this open for a while.

**Pull request merged**